build_projects/dotnet-cli-build/PublishTargets.cs

```diff
@@ -47,9 +48,9 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
         {
             if (CheckIfAllBuildsHavePublished())
             {
-                string targetContainer = $"{Channel}/Binaries/Latest/";
+                string targetContainer = $"{AzurePublisher.Product.Sdk}/{Channel}/";
                 string targetVersionFile = $"{targetContainer}{CliNuGetVersion}";
-                string semaphoreBlob = $"{Channel}/Binaries/publishSemaphore";
+                string semaphoreBlob = $"{targetContainer}/publishSemaphore";
                 AzurePublisherTool.CreateBlobIfNotExists(semaphoreBlob);
                 string leaseId = AzurePublisherTool.AcquireLeaseOnBlob(semaphoreBlob);
```

Comment:
We weren't doing it before...

I can add it, if you think that is the best approach.